sqlmesh/core/engine_adapter/bigquery.py

```diff
@@ -451,10 +455,14 @@ def _create_column_comments(
         logger.info(f"Registering column comments for table {table_name}")
         self._db_call(self.client.update_table, table=table, fields=["schema"])
 
-    def _build_description_property_exp(self, description: str) -> exp.Property:
+    def _build_description_property_exp(
+        self,
+        description: str,
+        trunc_method: t.Callable,
+    ) -> exp.Property:
         return exp.Property(
             this=exp.to_identifier("description", quoted=True),
-            value=exp.Literal.string(description),
+            value=exp.Literal.string(trunc_method(description)),
```

Comment:
Just confirming that `trunc_method` is correct here.

Reply:
Yep - this method handles both table and column comments, so I pass the correct `_truncate_XXX_comment` method to the `trunc_method` arg here